tdameritrade/auth/auth_form_executioner.py

```diff
@@ -0,0 +1,108 @@
+import time
+
+import requests_oauthlib
+
+
+class FormExecutioner:
+    tda_auth_url = 'https://auth.tdameritrade.com/auth'
+    tda_refresh_token_url = 'https://api.tdameritrade.com/v1/oauth2/token'
+
+    def __init__(
```

Comment:
does this work when no security questions are prompted?

Reply:
No and entire branch was reverted.